x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export class IndexPattern {
+  static readonly Events = 'events-endpoint-*';
+  static readonly Metadata = 'metrics-endpoint.metadata-*';
+  static readonly Policy = 'metrics-endpoint.policy-*';
+  static readonly Telemetry = 'metrics-endpoint.telemetry-*';
```

Comment:
@james-elastic @bradenlpreston @XavierM 
I've noticed that SIEM features allow the user to specify index patterns. Is it going to be weird if the endpoint stuff is hardcoded?
Is there any business case for allowing users to customize this?

Reply:
Allowing the user to configure the indices is a tricky problem. Currently the templates that define an index pattern for the mapping in elastic search is handled by the endpoint package. The ingest manager constructs the index pattern that is defined in the template based on the package that is being installed. That construction happens here: https://github.com/elastic/kibana/blob/master/x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/template.ts#L245

So we'd have to make it configurable in the ingest manager which is not possible at the moment. Otherwise the fields in the documents wouldn't have the right types and the data wouldn't be found using our queries.